packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3513,7 +3526,21 @@ class RenderSemanticsGestureHandler extends RenderProxyBox {
   @override
   void describeSemanticsConfiguration(SemanticsConfiguration config) {
     super.describeSemanticsConfiguration(config);
-
+    if (_semanticsActionOverrides != null && _semanticsActionOverrides.isNotEmpty) {
+      final Map<CustomSemanticsAction, VoidCallback> customAction = <CustomSemanticsAction, VoidCallback>{};
+      for (final CustomSemanticsAction override in _semanticsActionOverrides) {
+        switch (override.action) {
+          case SemanticsAction.scrollUp:
+          case SemanticsAction.scrollDown:
+          case SemanticsAction.scrollLeft:
+          case SemanticsAction.scrollRight:
+            customAction[override] = () { };
```

Comment:
This is odd, why are we setting this to an empty closure? Shouldn't these do something?

Reply:
There are two ways of doing this:
1. provide a callback with copying over the callback from semantics actions that this custom action overrides, and engine just call this custom action like regular custom action.

2. We don't provide the body here. the engine just grab the overrideId of this customaction and calls that action instead.

I choose to do 2 because that is less hustle in the framework side.

Reply:
We also need to worrying about keeping them in sync if we do (1)